site/assets/scss/_clipboard-js.scss

```diff
@@ -44,37 +37,19 @@
 
   // Boosted mod
   .bi {
-    width: 1.4em;
-    height: 1.4em;
+    width: 1.8125rem;
```

Comment:
I'd stick with `1.4em`. It is rather huge when displayed in the action bar next to the "Edit on StackBlitz" button.

Reply:
I agree that the size is way better but should we keep it even if they are a bit blurry ?